client/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@bible-engine/client",
-    "version": "1.0.2",
+    "version": "1.1.0-pre.1",
```

Comment:
this looks like a bit of an odd version (the "pre" part), maybe we can stick to semantic versioning?